### PR TITLE
chore(contributing): add PR naming and labelling conventions

### PR DESCRIPTION
**Pull request by GiovannaMonti**
### Description

This PR specifies PR submission rules in the CONTRIBUTING file and improves the PR submission template

### [MANDATORY] PR Checklist

Please make sure your PR complies with the following rules before submitting it.

#### PR conventions

- [x] PR title follows the `<type>(<scope>): <subject>` structure
- [x] The PR has been labeled according to the type of changes (e.g., enhancement, bug).

#### Additional code checks

Based on your changes, some of these checks may not apply. Please make sure to check the relevant items in the list.

- [ ] Changes are covered by tests 
- [ ] Changes to components are accessible and documented in the Storybook
- [ ] Typings have been updated
- [ ] New components are exported from the `index` file
- [ ] New files include the Apache 2.0 License disclaimer
- [ ] The browser console does not contain errors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ You can see the `npm` scripts available inside the `package.json` file and use t
 * Commits should follow the [Conventional Commit](https://www.conventionalcommits.org/) standard: `<type>(<scope>): <subject>` (e.g. `feat(core): Add new feature`);
 * At commit time, `yarn test` command script is automatically run. To skip this step, commit with `-n` flag.
 
+##### Pull Requests
```
Comment (fredmaggiowski) [inline CONTRIBUTING.md:25]: Does the PR template file include a notice about "making sure you've read the contributing guidelines, specifically for opening PR?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
-<!-- Hi, and thank you for your time dedicated to this pull request! -->
-
 ### Description
 
-<!-- Please provide a brief description of the work you have done and the motivations linked to these modifications. E.g.
+<!-- Please provide a brief description of your changes. Summarize the ratiomnale and impact on the codebase. E.g.
```
Comment (fredmaggiowski) [inline .github/pull_request_template.md:3]:
```suggestion
<!-- Please provide a brief description of your changes. Summarize the rationale and impact on the codebase. E.g.
```

**Review by fredmaggiowski — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -12,20 +10,28 @@
...
 
-<!-- Link to the issue, if present. E.g 
+<!-- Link to the issue, if present. E.g. 
     Closes #XYZ
 -->
 
-### Checklist
+### [MANDATORY] PR Checklist
+
+<!-- IMPORTANT: Before submitting the PR, please take a look at the CONTRIBUTING.md file for further details regarding standards and conventions adopted in this repository. You can find it here: https://github.com/mia-platform/design-system/blob/main/CONTRIBUTING.md -->
```
Comment (malta895) [inline .github/pull_request_template.md:19]: we could place it as the first checklist item?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -39,3 +44,5 @@ In order to tag the repository, use the following command:
 ```sh
 npm version <patch|minor|major>
```
Comment (malta895) [inline CONTRIBUTING.md:45]: isn't this outdated? we use `pnpm changeset` in this repo right?

Comment (GiovannaMonti) [reply CONTRIBUTING.md:45]: no, we still use npm version to tag

File: .github/pull_request_template.md (hunk 3 above)
Comment (GiovannaMonti) [reply .github/pull_request_template.md]: I made an update, see if it works better :)

Comment (malta895) [reply .github/pull_request_template.md]: yep looks good to me!

**Pull request merged**